lib/cmd/fh3/appdata/export/download.js

```diff
@@ -0,0 +1,50 @@
+"use strict";
+
+var fs = require('fs')
+  , fhreq = require('../../../../utils/request.js')
+  , common = require('../../../../common');
+
+module.exports = {
+  'desc': 'Download the result of a completed export job by job_id',
+  'examples': [{
+    cmd: [
+      'fhc addpata export download --appId=5ujx2eifvzaudq43nw4nmvcu --envId=live --jobId=5731da1dd3e2b283203c4054 ',
+      '--file=export_5ujx2eifvzaudq43nw4nmvcu.gz.tar'
+    ].join(''),
+    desc: 'Download the result of a completed export job by job_id'
+  }],
+  'demand': ['envId', 'appId', 'jobId', 'file'],
+  'alias': {},
+  'describe': {
+    'envId': 'Environment id',
+    'appId': 'Application id',
+    'jobId': 'Export job id',
+    'file': 'Path and filename of downloaded file'
+  },
+  'method': 'get',
+  'customCmd': function (params, cb) {
+    var url  = 'api/v2/appdata/' + params.envId + '/' + params.appId + '/export/data/' + params.jobId;
+
+    // Get the download URL
+    common.doApiCall(fhreq.getFeedHenryUrl(), url, null, 'Error downloading export', function (err, res) {
+      if (err) {
+        return cb(err);
+      }
+
+      params.url = res.url;
+      params.method = 'GET';
+      params.output = params.file;
+
+      // Check if the file already exists
+      fs.stat(params.file, function(err) {
+        if (!err) {
+          return cb("The file at path " + params.output + " already exists. Aborting.");
+        }
+
+        // Download it
+        console.log('Downloading from ' + res.url + '...');
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)